admin-dev/themes/new-theme/js/pages/product/components/dropzone/Dropzone.vue

```diff
@@ -364,13 +408,15 @@
         const selectedFile = this.selectedFiles[0];
 
         if (
-          this.coverData.file
+          coverData.file
```

Comment:
I think this `coverData` could be simplified now that it's handled in the DropzoneWindow:

We don't need to store the file and check if it matches, `saveSelectedFile` is only called when we save a single file, so there is necessarily only one selected so checking 
So checking `this.selectedFiles.length === 1 && coverData.file.image_id === selectedFile.image_id` is not necessary it will likely always be true

So the window component should simple have a boolean `isCover` in its data, which is bound to the checkbox by a `v-model` and pre-filled thanks to the watcher on `selectedFiles`

Reply:
But it would harder the future when we'll have to handle multiple caption save as `saveSelectedFile` will become `saveSelectedFiles` and handle multiple files

Reply:
Simplified it